### DISCUSSION TITLE
Any way to change the press action of the CPU plugin?

### DISCUSSION THREAD

**Discussion by lgander** [elgatosf/elgato#51] [Help]
I'd like to change it so it jumps to another page (instead of the current "open task manager" function).
Any ideas?
Thanks,

**Comment by GeekyEggo** [comment]
Hey @lgander, thanks for your feedback.

At the moment this isn't possible. You can however submit a feature request on our [streamdeck-cpu](https://github.com/elgatosf/streamdeck-cpu) repository. This helps the team gather insights into what features users would like to see when they're planning for future updates.
